src/seedu/addressbook/data/person/Email.java

```diff
@@ -58,4 +58,12 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString(Printable... printables) {
+		if (isPrivate) {
+			return "Email: (Private) ";
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.